Add API users' passwords maximum length to `Securing the API` section (#5056)
wazuh/wazuh-documentation: source/user-manual/api/securing-api.rst

    The default password can be changed using the following Wazuh API request: :api-ref:`PUT /security/users/{user_id} <operation/api.controllers.security_controller.update_user>`

    .. note::

>       The password for users must contain between 8 and 64 characters. It must also use at least one uppercase and one lowercase letter, a number, and a symbol.

Reviewer: ```suggestion
      The password for users must be between 8 and 64 characters long. It should contain at least one uppercase and one lowercase letter, a number, and a symbol.
```
